care/users/api/viewsets/change_password.py

```diff
@@ -0,0 +1,47 @@
+from rest_framework import serializers
+from django.contrib.auth import get_user_model
+from rest_framework import status
+from rest_framework.generics import UpdateAPIView
+from rest_framework.response import Response
+from rest_framework.permissions import IsAuthenticated
+
+User = get_user_model()
+
+class ChangePasswordSerializer(serializers.Serializer):
+    model = User
+
+    """
+    Serializer for password change endpoint.
+    """
+    old_password = serializers.CharField(required=True)
+    new_password = serializers.CharField(required=True)
+
+    def check_old_password(self, instance, validated_data):
+        if instance.check_password(validated_data.get("old_password")):
```

Comment:
can we just return this value, do we need an if-else here? also, why is this in the serializer?